CONTRIBUTING.md

```diff
@@ -33,27 +43,27 @@ For functions and methods, parameters are documented first and return type next.
 /// A happy little fish.
 class Fish : public Example {
     public:
-        /// Create new fish.
-        /**
-         * @param name The name to give the fish.
-         */
-        Fish(const std::string& name);
-        
-        /// Display the name of the fish.
-        /**
-         * @param prefix Whether to prefix the output with 'A fish called '.
-         */
-        void DisplayName(bool prefix = false) const;
-        
-        /// Get whether the fish is happy.
-        /**
-         * @return Whether the fish is happy.
-         */
-        bool GetHappy() const;
-        
+    /// Create new fish.
+    /**
+    * @param name The name to give the fish.
```

Comment:
Improper * alignment in doxygen comment. (Check if clang-format allows this or if it changes it back again. If it changes it back again, we may have to use a regex to ignore doxygen comments, or live with improperly formatted doxygen comments).